integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -3310,6 +3315,56 @@ msgid ""
 msgstr ""
 "Ob automatische Übersetzungen in Leichte Sprache mit SUMM.AI aktiviert sind"
 
+#: cms/models/regions/region.py
+msgid "DeepL credits renewal date"
+msgstr "DeepL Credits Zurücksetzungsdatum"
+
+#: cms/models/regions/region.py
+msgid "Day of the year on which the budget usage is reset"
+msgstr "Tag des Jahres zu welchem das Budget zurück gesetzt wird"
```

Comment:
There is a tiny typo here: zurückgesetzt is [one word](https://www.duden.de/rechtschreibung/zuruecksetzen)